assets/app/styles/_tile.less

```diff
@@ -5,11 +5,13 @@
 .tile {
   background: @console-panel-color;
   .panel_default_boxshadow();
-  padding: 10px 20px; 
+  padding: 10px 20px;
   margin-bottom: 20px;
   word-wrap: break-word;
```

Comment:
though, that might make normal words break halfway through instead of wrapping like we want... hiding overflow might be better